apix-impl/build.gradle

```diff
@@ -1,30 +1,39 @@
-import com.github.dynamicextensionsalfresco.gradle.configuration.BaseConfig
-import com.github.dynamicextensionsalfresco.gradle.tasks.InstallBundle
+plugins {
+    id 'eu.xenit.de' version '2.0.2' apply false
+    id 'eu.xenit.amp' version '0.2.1' apply false
+}
 
 subprojects {
-    // Isolate the version number ("apix-impl-52" -> "52").
-    def subproject_alfresco_version = project.name.substring(project.name.length() - 2)
-
     publishing {
         publications {
-            amp(MavenPublication) {
-                artifactId "apix-impl-${subproject_alfresco_version}-amp"
-                // TODO: This should be replaced by a proper reference to an artifact
-                artifact file(ampFile)
+            maven(MavenPublication) {
+                artifact tasks.amp
             }
         }
     }
 
     apply from: "${rootProject.projectDir}/publish.gradle"
     apply from: "${project.projectDir}/overrides.gradle"
-    apply plugin: 'org.dm.bundle'
-    apply plugin: 'ampde'
+    apply plugin: 'eu.xenit.de'
```

Comment:
Applying the `eu.xenit.de` plugin implicitly adds a dependency on `eu.xenit.de:annotations` and `eu.xenit.de:webscripts`, with the same version of the `eu.xenit.de` plugin unless an explicit version is specified in `alfrescoDynamicExtensions.versions.dynamicExtensions`.

This might not be what you expect, since the `de_version` variable is now ignored.